app/posts/apply-for-teacher-training/2021-09-22-degree-grade-requirement-warning-message.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Degree grade requirement warning message
+description: Warning candidates when they do not meet the degree grade requirement.
+date: 2021-09-22
+---
+
+One of the reasons why providers reject candidates is because they do not have the right degree grade (for example 2:1 or above).
+
+Each course has a different minimum degree requirement. While some courses accept any honours degree, most ask for a minimum grade.
```

Comment:
Is this true? I don't think I've seen any comprehensive analysis on this (as it was free text before)?